graphql-api/src/test/java/com/expediagroup/streamplatform/streamregistry/graphql/model/inputs/NameNormaliserTest.java

```diff
@@ -0,0 +1,39 @@
+package com.expediagroup.streamplatform.streamregistry.graphql.model.inputs;
+
+import static com.expediagroup.streamplatform.streamregistry.graphql.model.inputs.NameNormaliser.normalise;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+public class NameNormaliserTest {
+  @Test
```

Comment:
I'm not sure if  abc_1_def is acceptable or should cause an exception. Would be nice to see a test which clarifies and validates the correct outcome.

Reply:
Yes, should be acceptable - added a test